network-api/networkapi/wagtailpages/pagemodels/buyersguide/products.py

```diff
@@ -1026,7 +1029,24 @@ def localized_related_products(self):
         )
         related_products = localize_queryset(related_products, preserve_order=True)
         return related_products.specific()
-
+    
+    @property
+    def preview_related_products(self) -> list:
+        """
+        Fetches related product updates for CMS page previews.
+        """
+        related_products = orderables.get_related_items(
+            self.related_product_pages.all(),
+            "related_product",
+        )
```

Comment:
I've just noticed something: this **does not** guarantee to order the related products by their `sort_order`

![image](https://github.com/MozillaFoundation/foundation.mozilla.org/assets/61277874/c0754639-80b4-4d39-8b77-39d349994bc7)

Reply:
Thanks for the callout @jhonatan-lopes!

[Here is a link ](https://github.com/MozillaFoundation/foundation.mozilla.org/pull/11844/files#r1483822553)to the solution I have implemented to make sure that they are always sorted by sort_order.